### PR TITLE
Static pages editor

### PR DESCRIPTION
**Pull request by kstefanini**

### CODE REVIEW

File: templates/admin/edit-page.html.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+Édition de la page statique : {{ page.title }}
+{% endblock %}
+
+{% block main_content %}
+    {% include 'components/app-flashes.html.twig' %}
```
Comment (ya-web) [inline templates/admin/edit-page.html.twig:8]: app-flashes are now loaded in base.html.twig

File: src/Controller/PagesController.php [hunk 2]
```diff
@@ -41,14 +41,20 @@ public function index(
     }
 
     /**
-     * @Route("/apropos", name="apropos")
+     * @Route("/a-propos", name="a-propos")
```
[the remark this answers is not in the document]
Comment (kstefanini) [reply src/Controller/PagesController.php:44]: Yup, greped all files, should be fine

File: src/Controller/AdminController.php [hunk 3]
```diff
@@ -85,6 +87,66 @@ public function editSpeciesPage(
...
+            'category' => Post::CATEGORY_PAGE,
+            'slug' => $slug,
+        ]);
+        if (!$page) {
+            $page = new Post();
+            $page->setContent('');
+            $page->setAuthor($this->getUser());
+            $page->setCategory(Post::CATEGORY_PAGE);
+            $page->setTitle(BreadcrumbsGenerator::MENU[$slug]);
+            $page->setCreatedAt(new \DateTime());
```
[the remark this answers is not in the document]
Comment (kstefanini) [reply src/Controller/AdminController.php:128]: Could be, i'll plan to refactor all at once in another PR

File: templates/admin/edit-page.html.twig (hunk 1 above)
Comment (kstefanini) [reply templates/admin/edit-page.html.twig:8]: Oh nice!

**Pull request merged**